pact/marmalade.repl

```diff
@@ -906,6 +906,7 @@
 (expect "OFFER events"
    [{"name": "marmalade.ledger.OFFER","params": ["project-2" "bob" 0.2 120]}
     {"name": "marmalade.ledger.SALE","params": ["project-2" "bob" 0.2 120 (pact-id)]}
+    {"name": "marmalade.fixed-quote-policy.QUOTE","params": ["gpplCkSYRYPuMk0HbK_KogMFhhYYVaHuawOOHuNoRVw" "project-2" (read-msg "quote")]}
```

Comment:
Can we replace `gpplCkSYRYPuMk0HbK_KogMFhhYYVaHuawOOHuNoRVw` with `(pact-id)` for consistence? Same with the following ones